docs/observability/metrics.md

```diff
@@ -135,6 +135,18 @@ Total VM filesystem capacity in bytes. Type: Gauge.
 ### kubevirt_vmi_filesystem_used_bytes
 Used VM filesystem capacity in bytes. Type: Gauge.
 
+### kubevirt_vmi_guest_hostname_info
+Guest hostname reported by the guest agent. The value is always 1. Type: Gauge.
+
+### kubevirt_vmi_guest_load_15m
+Guest system load average over 15 minutes as reported by the guest agent. Type: Gauge.
```

Comment:
What does 'load' mean in this case?

Reply:
Please update this info in the metrics description

Reply:
guest system load is the more accurate description as used the kernel